packages/xo-server-sdn-controller/src/ovsdb-client.js

```diff
@@ -438,9 +436,7 @@ export class OvsdbClient {
 
   async _sendOvsdbTransaction(params, socket) {
     const stream = socket
-
-    const requestId = this._requestID
-    ++this._requestID
+    const requestId = ++this._requestID
```

Comment:
Idem.

Reply:
Not a pb, the index will start at 1 instead of 0.
Nothing else changes.